lib/openscap_parser/set_values.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'openscap_parser/set_value'
+
+module OpenscapParser
+  module SetValues
```

Comment:
Ok, so I think for this and other 'Selectors' (section 6.5.3 of the [XCCDF spec](https://csrc.nist.gov/publications/detail/nistir/7275/rev-4/final)) we should namespace them to reduce confusion. In almost all cases, we will want to be able to parse any of the selectors, not just `set-value`. What do you think about an `OpenscapParser::Selectors` module that includes `SetValues`? We don't need to implement the other selectors such as refine-value or refine-rule until we need them, but it allows us to add them easily later without changing the design.